src/templating/data.ts

```diff
@@ -131,3 +136,18 @@ export async function getTemplateFiles(
     throw new Error('Invalid template');
   }
 }
+
+async function buildHeaders(): Promise<OutgoingHttpHeaders> {
+  let githubToken = '';
+  const { localEnv } = await readLocalEnvFile({ cwd: process.cwd() });
```

Comment:
I noticed typical usage of `readLocalEnvFile` included the cwd and env from flags. Should I have also done the same?

Reply:
I would actually use `process.env.TWILIO_SERVERLESS_GITHUB_TOKEN` instead of reading the local env file for this. Using `readLocalEnvFile` would enable people to place this into `.env` files but that would also cause that variable to be uploaded to Twilio Functions if they ran `twilio-run deploy`. Instead I think this token should live on your dev machine only. So using `process.env` is the right approach here.